Minesweeper.py

```diff
@@ -43,68 +28,86 @@
                 print(board[y][x], end=" ")
         print()
 
-    # Print the game stats
+def print_stats():
     print("Moves:", moves)
     print("Flags:", flags)
     print("Mines found:", mines_found)
 
-    # Get the player's input
-    action = input("Enter your action (move, flag, or unflag): ")
-    direction = input("Enter the direction (up, down, left, or right): ")
-
-    # Update the player's position
+def update_player_pos(direction):
+    x, y = player_pos
```

Comment:
Consider checking that the new player position does not move outside the bounds of the board after updating the coordinates. ```diff -46:         x += 1 +46:         if 0 <= x < BOARD_SIZE and 0 <= y < BOARD_SIZE: ``` ---